.circleci/config.yml

```diff
@@ -139,6 +181,10 @@ jobs:
           keys:
             - chronograf-npm-packages-{{ checksum "ui/package-lock.json" }}
       - run: make build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - bin/linux/influxd
```

Comment:
Perhaps also persist bin/linux/influx

Reply:
Added `- bin/linux/influx` to be shared between jobs in a workspace